api/v1beta1/disruption_webhook.go

```diff
@@ -44,6 +46,11 @@ func (r *Disruption) ValidateCreate() error {
 		return errors.New("the controller is currently in delete-only mode, you can't create new disruptions for now")
 	}
 
+	// handle a disruption using the onInit feature without the handler being enabled
+	if !handlerEnabled && r.Spec.OnInit {
+		return errors.New("the chaos handler is disabled but the disruption onInit field is set to true, please enable the handler by specifying the --handler-enabled flag to the controller if you want to use the onInit feature")
```

Comment:
I'd argue this error message should help the user understand that (almost certainly) the handler is disabled because the current cluster is too old of a k8s version. Maybe appending something like "if you are creating this disruption on a k8s cluster < 1.15, then this is disabled intentionally as it is not supported"